packages/sui-ssr/test/server/utilsSpec.js

```diff
@@ -1,27 +1,15 @@
 import {expect} from 'chai'
-import {publicFolder, __RewireAPI__ as utilsRewireAPI} from '../../server/utils'
-import {getMockedRequest, multiSiteMapping} from './fixtures'
+import {publicFolder} from '../../server/utils'
+import {getMockedRequest} from './fixtures'
+import {publicFolderWithMultiSiteConfig} from './fixtures/utils'
 
 describe('[sui-ssr] Utils', () => {
   describe('Public folder', () => {
     describe('In a multi site project', () => {
-      beforeEach(() => {
-        utilsRewireAPI.__Rewire__('multiSiteMapping', multiSiteMapping)
-        utilsRewireAPI.__Rewire__(
-          'multiSiteKeys',
-          Object.keys(multiSiteMapping)
-        )
-        utilsRewireAPI.__Rewire__('isMultiSite', true)
-      })
-
-      afterEach(() => {
-        utilsRewireAPI.__ResetDependency__('multiSiteMapping')
-        utilsRewireAPI.__ResetDependency__('multiSiteKeys')
-        utilsRewireAPI.__ResetDependency__('isMultiSite')
-      })
-
       it('Should build the public folder name properly', () => {
-        const folderName = publicFolder(getMockedRequest('www.bikes.com'))
+        const folderName = publicFolderWithMultiSiteConfig(
```

Comment:
This specs tests `utilsFactory` behaviour but, given it is being imported in `test/server/fixtures/utils`, and then this file imports its execution `import {publicFolderWithMultiSiteConfig} from './fixtures/utils'`,
it looks like if we want to test a fixture function instead `utilsFactory`
https://github.com/SUI-Components/sui/blob/1ce6322b9a4da1849f28f7886e8b2f7ef95f7eaf/packages/sui-ssr/test/server/fixtures/utils.js#L4